src/Helpers/ObjectHelperTrait.php

```diff
@@ -219,4 +268,30 @@ public static function parseManifest(string $manifest): array
 
 		return $result;
 	}
+
+	/**
+	 * Get current url with params.
```

Comment:
```suggestion
	 * Get current URL with params.
```
@goranalkovic-infinum you've missed this one 🤣